bootstrap/src/main/java/org/jboss/windup/bootstrap/commands/windup/RunWindupCommand.java

```diff
@@ -129,7 +131,7 @@ private void runWindup(List<String> arguments)
             }
             else if (Boolean.class.isAssignableFrom(option.getType()))
             {
-                optionValues.put(option.getName(), true);
+                optionValues.put(option.getName(), !"false".equalsIgnoreCase(argument));
```

Comment:
It should still do that. The intent is:
--foo -> TRUE
--foo=bar -> TRUE
--foo=false -> FALSE
[not present] -> FALSE